src/main/proto/streams.proto

```diff
@@ -0,0 +1,125 @@
+syntax = "proto3";
+package io.axoniq.axonserver.grpc.streams;
+import "event.proto";
+import "google/protobuf/empty.proto";
+option java_multiple_files = true;
+
+/* Service providing operations for persistent event streams, event streams where Axon Server keeps
+   track of the progress. All operations require a header (AxonIQ-Context) to be passed with each request to define
+   the context.
+ */
+service EventStreamService {
```

Comment:
Suggestion: rename to PersistentStreamsService
I also suggest to change the name of the proto file to persistent-streams.proto